### PR TITLE
Add steps to create a fork

### PR DESCRIPTION
**Pull request by ericrommel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,3 +21,49 @@
 
 ### [Week 9](weeks/week09/week09.md): Advanced AI-Driven Automation Techniques
 
+### [Week 10](weeks/week10/week10.md): Building a Test Automation Suite, utilizing both web and API testing
+
+
+## Fork the Repo
+
+A `fork` is a new repository that shares code and visibility settings with the original “upstream” repository. This is a good way to have your own repository and still having updates from the "upstream". Here the basic steps to take it. Follow the references to you want to get it deeper.
```
Comment (ksauhen) [inline README.md:29]: Follow the references **if** you want to get it deeper.

File: README.md [hunk 2]
```diff
@@ -18,3 +21,49 @@
...
+```bash
+$ git checkout main
+$ git remote add upstream the-clone-https-url
+$ git pull upstream main
+$ git push origin main
+```
+
+Commands explained:
+ - `git checkout main`: It is to switch to the forked `main` branch.
+ - `git remote add upstream the-clone-https-url`: It helps you to create a label called `upstream` that is connected to the URL for the `main` project. This is a one-time configuration. Once done, you don't need to it again next time.
```
Comment (ksauhen) [inline README.md:59]: Once done, you don't need to **do** it again next time.

File: README.md (hunk 1 above)
Comment (ericrommel) [reply README.md]: Fixed!

File: README.md (hunk 2 above)
Comment (ericrommel) [reply README.md]: Fixed!

**Review by ksauhen — APPROVED** [review]

**Pull request merged**